src/_TextInputWrapper.tsx

```diff
@@ -27,16 +27,22 @@ const sizeVariants = variant({
   prop: 'size',
   variants: {
     small: {
+      '--inner-action-size': '20px',
       minHeight: '28px',
       px: 2,
       py: '3px',
       fontSize: 0,
       lineHeight: '20px',
     },
+    medium: {
+      '--inner-action-size': '24px',
+    },
     large: {
+      '--inner-action-size': '28px',
       px: 2,
       py: '10px',
-      fontSize: 3,
+      lineHeight: '0',
```

Comment:
Removing this line-height doesn't seem to have any affect on leading visuals or trailing visuals/actions. 🔪 it?

```suggestion
```